ivy_tests/test_ivy/test_functional/test_extensions/test_core/test_manipulation.py

```diff
@@ -414,3 +414,42 @@ def test_top_k(
         largest=largest,
         out=None,
     )
+
+
+# fliplr
+@handle_cmd_line_args
+@given(
+    dtype_and_m=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=-100,
+        max_value=100,
+        min_num_dims=2,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
```

Comment:
```suggestion
        min_num_dims=2,
```
I don't think we need all of those constraints except the `min_num_dims` here. Can you verify if they are indeed required here?

Reply:
Yes, you are correct. I've committed the suggested change.